src/content/docs/en/reference/adapter-reference.mdx

```diff
@@ -394,3 +395,74 @@ function createEdgeMiddleware(middlewareEntryPoint) {
     // emit a new physical file using your bundler
 }
 ```
+
+### envGetSecret
+
+This is a feature to allow your adapter to retrieve secrets configured by users in `env.schema`.
+
+Enable the feature by passing any valid `SupportsKind` value to the adapter:
```

Comment:
```suggestion
Enable the feature by passing `envGetSectret: 'stable'` to the `adapterFeatures`:
```
Does this make more sense now?

Reply:
Well technically people could say it's in experimental, that's why i didn't want to say stable here

Reply:
This is for the 5.0 beta docs though right? It's not experimental in these docs?

Reply:
I mean that's a different thing. Yes `astro:env` is stable but this option is for the adapter to tell its users if it supports `astro:env` well

Reply:
AH, I was missing that part. OK, makes sense then!